pkg/liqo-controller-manager/foreign-cluster-operator/permission.go

```diff
@@ -47,12 +47,6 @@ func (r *ForeignClusterReconciler) ensurePermission(ctx context.Context, foreign
 			klog.Error(err)
 			return err
 		}
-		if r.OwnerReferencesPermissionEnforcement {
-			if err = r.NamespaceManager.UnbindIncomingClusterWideRole(ctx, remoteCluster); err != nil {
-				klog.Error(err)
-				return err
-			}
-		}
```

Comment:
Is this change necessary? (the same for the binding)

They are managing the cluster role binding that I think is not related to the tenant resource

Reply:
It seems to me the ClusterRole is related to the tenant finalizer only, if I haven't misinterpreted something. 
https://github.com/liqotech/liqo/blob/fadcbcd767b7d60959d3a53ee311d1cf91bdd4a2/pkg/tenantNamespace/clusterwide.go#L49-L56

Reply:
BTW, do the E2E tests run with the strict permissions mode?

Reply:
ok, yes you are right 👍

no, the e2e are running with the default mode (if I remember correctly). We should change it